### ISSUE TITLE
Is there a repo for scheme-ts-mode for emacs ?

### ISSUE THREAD

**Issue by ieugen** [6cdh/tree-sitter-scheme#9]
Hi, 

Given this grammar for scheme and emacs 21.1 with tree-sitter support, is there an emacs mode that uses this?

It would be nice to be mentioned if that is the case.
If not, are there any plans to add / support one?

Thanks

**Comment by 6cdh** [comment]
No, I don't know. I'm not familiar with scheme-ts-mode.

**Comment by ieugen** [comment]
I don't think it exists, but what I am asking about is an emacs tree-sitter (ts) mode that  for scheme that uses this grammar.

**Comment by 6cdh** [comment]
I don't use Emacs. Let's waiting for someone do it and solve this issue.

**Comment by dieggsy** [comment]
This seems somewhat doable for me though it'll probably take me a good while to learn all the ins and outs of the tree sitter font locking query rules/syntax

**Comment by dieggsy** [comment]
Here's a proof of concept, still missing several keywords and indentation rules (and of course proper packaging):

https://git.sr.ht/~dieggsy/scheme-ts-mode

(Currently this is using highlight-numbers-number for the number face)

**Comment by ieugen** [comment]
@dieggsy : Thanks for working on that. 
I wanted to open an issue or get in touch in sr.ht and could not so I will post here.

I'm using your package in emacs (new user) using use-package and straight, but I get some errors for this definition, which I adapted from someplace else. 
```
(use-package scheme-ts-mode
  :straight '(scheme-ts-mode
              :type git
              :repo "https://git.sr.ht/~dieggsy/scheme-ts-mode"))
```
I don't do a lot of Scheme recently, but it has an appeal and would love to see tooling improve. 
So I am willing to test things and report issues. 
If you would like to get reports and other help, please let people know what and how they can do that. 

```
 ■  Error (use-package): scheme-ts-mode/:catch: Loading file /home/ieugen/.config/emacs/straight/build/scheme-ts-mode/scheme-ts-mode.elc failed to provide feature ‘scheme-ts-mode’
⛔ Error (use-package): scheme-ts-mode/:catch: Loading file /home/ieugen/.config/emacs/straight/build/scheme-ts-mode/scheme-ts-mode.elc failed to provide feature ‘scheme-ts-mode’
⛔ Error (use-package): scheme-ts-mode/:catch: Loading file /home/ieugen/.config/emacs/straight/build/scheme-ts-mode/scheme-ts-mode.elc failed to provide feature ‘scheme-ts-mode’
```

**Comment by ieugen** [comment]
Thanks @dieggsy . Will update the project.
I will close this issue since now there is a repo for scheme-ts-mode .
